### ISSUE TITLE
Namespace conflict resolution

### ISSUE THREAD

**Issue by AndreyZM** [bridgedotnet/Bridge#1814]
### Steps To Reproduce
http://deck.net/0f11241e576543a1a1146bf528999a0e
```csharp
using ContentManagerBase = content.ContentManagerBase;

namespace achievements.content
{
	public class A{}//Need to show result in Deck.net
    public class ContentAchievements : ContentManagerBase
    {
    	public void Method()
        {
        	//Invalid js code here
        	var a = PNG;
        }
    }
   }
   
namespace content
{
	public class B:achievements.content.A{}//Need to show result in Deck.net
    public class ContentManagerBase
    {
        protected const string PNG = ".png";
    }
}
```

```js
Bridge.assembly("Demo", function ($asm, globals) {
    "use strict";

    Bridge.define("achievements.content.A");

    Bridge.define("achievements.content.ContentAchievements", {
        inherits: [content.ContentManagerBase],
        method: function () {
            //Invalid js code here
            var a = [NamespaceResolveResult [MergedNamespace achievements.content (from 1 assemblies)]].contentManagerBase.pNG;
        }
    });
});

```

**Comment by LeonidVeriga** [comment]
Hi @AndreyZM, thanks for the nice case!

We investigated the issue and found there are some things related to NRefactory to fix this issue.
The fix would be too risky to include into 15.0.1. We will do our best to fix it in 15.0.

Would you mind using a workaround:

[Deck](http://deck.net/737718697eadd92b14b712022ecb3f4b)
```cs
var a = PNG;
```
 -->
```cs
var a = global::content.ContentManagerBase.PNG;
```

**Comment by AndreyZM** [comment]
Sad, because i search for "[NamespaceResolveResult" in compiled js and found 60 places.

**Comment by geoffreymcgill** [comment]
This should be fixed by Monday (if not sooner) in Bridge 5.1.0.

**Comment by geoffreymcgill** [comment]
A pull request has been submitted by the team with a fix for this issue. 

https://github.com/bridgedotnet/Bridge/pull/1837

We will do our best to integrate this pr into the master branch as soon as possible.

**Comment by geoffreymcgill** [comment]
This issue has been fixed and included in the Bridge 15.0.2 release that was pushed to NuGet. 

It will also be included in the upcoming Bridge 15.1.0 release. 

Your original Deck now compiles and runs correctly. 

Hope this helps.
